src/components/dashboard/Restaurants.vue

```diff
@@ -2,6 +2,48 @@
   <div> 
       <h2>Content from restaurants component </h2>
       <p> {{ restaurants }}</p>
+     
```

Comment:
Fix indentation please.
We have to find a way to make the IDE fix indentation automatically for Vue Templates